Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.h

```diff
@@ -0,0 +1,46 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class ConversationViewItem;
+
+typedef NS_ENUM(NSUInteger, OWSMessageGestureLocation) {
+    // Message text, etc.
+    OWSMessageGestureLocation_Default,
+    OWSMessageGestureLocation_OversizeText,
+    OWSMessageGestureLocation_Media,
+    OWSMessageGestureLocation_QuotedReply,
+};
+
+@interface OWSMessageBubbleView : UIView
```

Comment:
Pulled out `OWSMessageBubbleView` that we can use in 

* conversation view cells
* message detail view.